lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +855,41 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+/// Check if the input of this method (vector of NodeValues) can be 
+/// legal input of CreateConcat function.
+/// \returns the dimension where the Concat will take place. If not 
+/// concat-able, return -1.
+size_t checkInputsForConcat(std::vector<NodeValue> inputs) {
```

Comment:
declare either as a static method or in an anonymous namespace (static is more common in our codebase)

Reply:
done